code/modules/battlemonsters/items/card.dm

```diff
@@ -179,3 +179,22 @@
 	else
 		SSbattlemonsters.ExamineMonsterCard(user,prefix_datum,root_datum,suffix_datum)
 
+/obj/item/battle_monsters/card/MouseEntered(location, control, params)
+	. = ..()
+	if(!facedown || Adjacent(usr))
```

Comment:
does adjacent even work if the item's in your contents?

Reply:
worked in testing, yep